.github/workflows/CI.yml

```diff
@@ -285,11 +285,17 @@ jobs:
           key: npm-cache-test-linux-aarch64-gnu-${{ matrix.node }}-${{ hashFiles('package-lock.json') }}
       - name: Install dependencies
         run: npm clean-install --ignore-scripts
+      - name: Set up QEMU
```

Comment:
Got around the missing docker image in napi-rs for Node 20 by using QEMU with a standard Node 20 image.

Reply:
is this setup incompatible with Node 12?

Reply:
Nope. I've brought it back now. I'll file a follow up ticket to remove Node 12 support as all Github actions using Node 12 are now deprecated. (https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/)